assets/js/base/components/quantity-selector/index.js

```diff
@@ -93,10 +97,10 @@ const QuantitySelector = ( {
 					'woo-gutenberg-products-block'
 				) }
 				className="wc-block-quantity-selector__button wc-block-quantity-selector__button--minus"
-				disabled={ disabled || currentValue <= 0 }
+				disabled={ disabled || ! canDecrease }
 				onClick={ () => {
-					const newQuantity = currentValue - 1;
-					setCurrentValue( newQuantity );
+					const newQuantity = quantity - 1;
+					onChange( newQuantity );
```

Comment:
No need for constant defined here (realize you didn't do in this pull):

```suggestion
					onChange( quantity - 1 );
```

Reply:
See above ⬆

Reply:
Or below, depending on comment order ↘️